docs: add engagement rules (#34)
PyTorchLightning/ecosystem-ci: .github/PULL_REQUEST_TEMPLATE.md

- [ ] Was this discussed/approved via a GitHub issue? (no need for typos and docs improvements)
- [ ] Did you create/update your **configuration file**?
- [ ] Did you **set `runtimes` in config** for GitHub action integration?

> - [ ] Did you **add your config to CI** in Azure pipeline for 100\* stars?

Reviewer: ```suggestion
- [ ] Did you **add your config to CI** in Azure pipeline (only projects with 100+ github stars)?
```

> - [ ] Did you **set `runtimes` in config** for GitHub action integration?
> - [ ] Did you **add your config to CI** in Azure pipeline (only projects with 100+ github stars)?

Reviewer 2: ```suggestion
- [ ] Did you **set `runtimes` in config** for GitHub action integration?
- [ ] Did you **add your config to CI** in Azure pipeline (only projects with 100+ GitHub stars)?
```

- [ ] Are all integration **tests passing**?

## What does this PR do? \[optional\]

> Fixes # (issue) \[add link to created issue in your repository if any\]
> OR link your project for clarity.

Reviewer 2: ```suggestion
Fixes # (issue) \[add a link to the created issue in your repository if any\]
OR link your project for clarity.
```
